ludwig/encoders/bag_encoders.py

```diff
@@ -64,8 +65,7 @@ def __init__(
             embeddings_on_cpu=embeddings_on_cpu,
             dropout=dropout,
             embedding_initializer=weights_initializer,
-        )
-
+        ).to(DEVICE)
```

Comment:
Not sure if you meant to leave this in.

Could you also remove the `DEVICE` initialization at line 28?

Reply:
Ah, thanks!